### PR TITLE
Attempt to fix Claim protection

### PR DESCRIPTION
**Pull request by corytortoise**
This PR is my attempt to fix issue #1. I have not tested this. If anyone has the time, I would greatly appreciate somebody else testing it. Otherwise, I will do it ASAP.

### CODE REVIEW

File: src/FactionsPro/FactionListener.php [hunk 1]
```diff
@@ -97,8 +97,10 @@ public function factionPVP(EntityDamageEvent $factionDamage) {
 		}
 	}
 	public function factionBlockBreakProtect(BlockBreakEvent $event) {
-		if($this->plugin->isInPlot($event->getPlayer())) {
-			if($this->plugin->inOwnPlot($event->getPlayer())) {
+       $x = $event->getBlock()->getX();
+       $z = $event->getBlock()->getZ();
+		if($this->plugin->pointIsInPlot($x, $z)) {
+			if($this->plugin->factionFromPoint($x, $z) == $this->plugin->getFaction($event->getPlayer()) {
```
Comment (Awzaw) [inline src/FactionsPro/FactionListener.php]: getFaction() takes the players name not a Player object, and there's missing brackets for the if statement

**Comment by corytortoise** [comment]
Okay, I'll fix this when I get a chance.

File: src/FactionsPro/FactionListener.php (hunk 1 above)
Comment (corytortoise) [inline src/FactionsPro/FactionListener.php]: Did you mean parentheses instead of bracket? I found a missing parentheses, but brackets seem correct to me.

Comment (SOF3) [inline src/FactionsPro/FactionListener.php]: and the indents :D

Comment (Awzaw) [inline src/FactionsPro/FactionListener.php]: Where I'm from we call () brackets, and [] square brackets ;-)

**Comment by Awzaw** [comment]
You should really test before submitting a PR - getFaction() takes the players name, not a Player object. Not entirely your fault as the code is not exactly consistent, but you'd have spotted that if you had tested it...

**Comment by corytortoise** [comment]
Okay, good points. I'll keep these things in mind.

**Pull request merged**